src/exportToSheets.ts

```diff
@@ -95,4 +89,13 @@ export async function exportToSheets() {
       ],
     },
   });
+
+  // then clear the excess data
+  const start = values.length + 1;
+  const end = start + 10000;
+
+  await sheets.spreadsheets.values.clear({
+    spreadsheetId,
+    range: `A${start}:Z${end}`,
```

Comment:
If I'm reading this correctly, we assume the new values will be at the head of the list, and the excess values will be any after the first N values.

Once concern is, are we sure that the insertion into sheets isn't eventually consistent?

Reply:
Yes, it batchUpdates with A1